test/test.cmake

```diff
@@ -1,5 +1,5 @@
 # ######################################################################################################################
-# Copyright (C) 2017 Advanced Micro Devices, Inc.
+# Copyright (C) 2023 Advanced Micro Devices, Inc.
```

Comment:
2017-2023

Reply:
Done. (been done for a while, just didn't mark here)